bin/i18n/yarn.lock

```diff
@@ -0,0 +1,543 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
note that the packages in `bin/i18n` are already managed with https://github.com/code-dot-org/code-dot-org/blob/staging/bin/i18n/package-lock.json

Reply:
oops! committed this because I got a warning on commit. I'll remove it.